<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controlpanel</class>
 <widget class="QWidget" name="Controlpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controlpanel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(39, 169, 126);
}
</string>
  </property>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数据可视化</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Cl_CheckBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Cl-</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Na_CheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Na+</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Cu_CheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>59</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Cu2+</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="SO4_CheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>67</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>SO42-</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Client_list">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Client_label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>客户端IP</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Recv">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>380</y>
     <width>391</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
background-color: rgb(46, 255, 221);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>实时通信(接收)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>640</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数据保存</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>690</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="LED4_label">
      <property name="text">
       <string>LED4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LED3_label">
      <property name="text">
       <string>LED3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LED2_label">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>LED2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LED1_label">
      <property name="text">
       <string>LED1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LED显示：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Rate_spinBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>调节频率</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(72, 0, 217);
font: 20pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="text">
    <string>离子色谱仪</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
